playwright/e2e/crypto/crypto.spec.ts

```diff
@@ -468,6 +469,12 @@ test.describe("Cryptography", function () {
 
             /* log out, and back in */
             await logOutOfElement(page);
+            // Reload to work around a Rust crypto bug where it can hold onto the indexeddb even after logout
+            // https://github.com/element-hq/element-web/issues/25779
+            await page.addInitScript(() => {
+                window.localStorage.clear();
+            });
```

Comment:
why do we need to clear the localstorage?

Reply:
The playwright `user` fixture injects credentials (inc a now-expired access token) into LS on reload, so this also clears them. The alternative would be a large refactor of this test but given its just for a workaround that seems silly

Reply:
There's no method at this time to remove an init script https://github.com/microsoft/playwright/issues/29499

Reply:
ah right. Stick that in a comment in the code, maybe?

(I wonder if there is an easy way to make that initscript only run once... but you're right, it's a distraction)